prov/efa/src/efa_hmem.h

```diff
@@ -36,6 +36,7 @@
 
 struct efa_hmem_info {
 	bool initialized; 	/* do we support it at all */
+	bool p2p_required;	/* Does the iface require p2p? */
```

Comment:
The name will confused with FI_HMEM_P2P_REQUIRED,  which is a user option.

This flag should should be `p2p_required_by_implementation`

Reply:
Ack, will change